lte/gateway/python/magma/enodebd/rpc_servicer.py

```diff
@@ -57,9 +59,9 @@ def _get_handler(self, device_serial: str) -> EnodebAcsStateMachine:
         return self.state_machine_manager.get_handler_by_serial(device_serial)
 
     def GetParameter(
-        self,
-        request: GetParameterRequest,
-        context: Any,
+            self,
```

Comment:
This format change appears to me it should fail GitHub Actions regarding python formatting standards - I see you are a first time contributor and 19 of our workflows in GH had not run so I just enabled them / you may now get more Linting findings to deal with FYI.

Reply:
Edit: Actually Python format check passed.  Was this file out of spec to start with? Any idea what's going on here @themarwhal ?

Reply:
I took a look, and I think our formatting script is missing E1 (indent styling) (see https://pypi.org/project/autopep8/). So this PR is good as far as our formatting standards goes. It might be good to enhance our formatting script / check to add the indent styling though.